scheduler/classes/WSRange.py

```diff
@@ -21,3 +22,108 @@ def getA1(self) -> str:
         """
 
         return self.start.getA1() + ":" + self.end.getA1()
+
+
+class CarBlock:
```

Comment:
Splitting this up into a CarBlock seems to make a lot of sense, I like this addition